crates/pipeline_manager/src/bin/pipeline-manager.rs

```diff
@@ -77,15 +77,14 @@ async fn main() -> anyhow::Result<()> {
     db.run_migrations().await?;
     let db = Arc::new(Mutex::new(db));
     let db_clone = db.clone();
-    let compiler_config_clone = compiler_config.clone();
     let _compiler = tokio::spawn(async move {
-        Compiler::run(&compiler_config_clone, db_clone)
+        Compiler::run(&compiler_config.clone(), db_clone)
             .await
             .unwrap();
     });
     let db_clone = db.clone();
     let _local_runner = tokio::spawn(async move {
-        local_runner::run(db_clone, &local_runner_config, &compiler_config).await;
+        local_runner::run(db_clone, &local_runner_config.clone()).await;
```

Comment:
Same question, are these clones required?

Reply:
Must have inherited this with reverting some of the JIT commits.